Dockerfile

```diff
@@ -7,7 +7,7 @@ WORKDIR /go/src/preflight
 RUN make build
 
 # ubi-minimal:latest image created 2021-06-22T13:04:30.956781
```

Comment:
I added this comment originally to make it easier to determine what/when the digest was for, but up to everyone if we want to keep this, as I can see it getting out of date or forgotten when we upgrade the version we are using here.

Reply:
Removed for now.